internal/cli/token.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"context"
+
+	"github.com/mongodb/mongocli/internal/validate"
+)
+
+// RefreshAndValidate a token from the current configuration.
+func RefreshAndValidate(ctx context.Context) error {
```

Comment:
[q] refresh an validate what?

Reply:
oh, right! updating to RefreshAndValidateToken